app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -33,7 +34,9 @@ class SearchLocationFragment : Fragment() {
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
         rootView = inflater.inflate(R.layout.fragment_search_location, container, false)
 
-        Utils.showSoftKeyboard(context, view)
+        val inputManager: InputMethodManager = context?.getSystemService(Context.INPUT_METHOD_SERVICE)
+            as InputMethodManager
+        inputManager.toggleSoftInput(InputMethodManager.SHOW_IMPLICIT, 0)
```

Comment:
Use the util method

Reply:
Sorry, but it doesn't work with Utils method, that method was used before but keyboard wasn't displayed 1st time the fragment was created. So I had to replace that with this.

Reply:
Then move to Util

Reply:
Okay